media-gfx/hydrus/hydrus-441.ebuild

```diff
@@ -0,0 +1,154 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
```

Comment:
Just drop Python 3.6 at this point. Able to do 3.9?

Reply:
Unfortunately QtPy still doesn't have 3.9. After that, it'll probably work with 3.9 fine.